pkgs/development/python-modules/simpy/default.nix

```
@@ -9,10 +10,17 @@ buildPythonPackage rec {
     sha256 = "dd8c16ca3cff1574c99fe9f5ea4019c631c327f2bdc842e8b1a5c55f5e3e9d27";
   };
 
+  nativeBuildInputs = [ setuptools_scm setuptools ];
+
+  propagatedBuildInputs = [ setuptools ];
+
+  checkInputs = [ pytestCheckHook ];
+
   meta = with lib; {
-    homepage = "https://simpy.readthedocs.io/en/latest/";
-    description = "A process-based discrete-event simulation framework based on standard Python.";
+    homepage = "https://simpy.readthedocs.io/en/${version}/";
+    description =
+      "A process-based discrete-event simulation framework based on standard Python.";
```

make it fit on one line. And i believe description generally doesn't include a period
```suggestion
    description = "Process-based discrete-event simulation framework based on standard Python";
```

but don't feel strongly about this one.

---

Definitely is more consistent this way!